Dockerfile-ubuntu2004

```diff
@@ -24,7 +24,10 @@ RUN apt-get update && apt-get -y install \
   wget \
   xz-utils \
   net-tools \
-  tcpreplay
+  tcpreplay \
+  python3 \
+  python3-venv \
+  python3-dev
```

Comment:
What is `python3-dev`? Is it required to run venv?

Reply:
If any pip packages require building the binary, then this is necessary. It is needed for Alpine, but I added it for Ubuntu just in case.

Reply:
ok 👍